sdk/servicebus/service-bus/migrationguide.md

```diff
@@ -33,8 +33,19 @@ We now have methods with similar names, signature and location to create senders
 We have a variety of new features in the version 7 of the Service Bus library.
 
 - A new `ServiceBusAdministrationClient` to perform operations like create/get/list/update/delete on queues/topics/subscriptions/rules. These were already available as part of a separate package `@azure/arm-servicebus` that uses Azure Resource Manager APIs but had the drawback of not supporting connection strings.
+- Ability to create a batch of messages with the smarter `ServiceBusSender.createBatch()` and `ServiceBusMessageBatch.tryAddMessage()` APIs. This will help you manage the messages to be sent in the most optimal way.
 - Ability to configure the retry policy used by the operations on the client, sender and receivers.
-- Ability to cancel async operations on the client, sender and receivers using the abort signal from `@azure/abort-controller`
+- Ability to cancel async operations on the client, sender and receivers and the management operations using the abort signal from `@azure/abort-controller`.
+- Ability to perform tracing with `[@azure/core-tracing](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/core/core-tracing/README.md)`
```

Comment:
Lets not call out tracing. It is something we consider to be in beta

Reply:
```suggestion
```